src/imaginghelper.js

```diff
@@ -689,8 +689,17 @@
      **/
     function onOpen() {
         this._haveImage = true;
-        this.imgWidth = this._viewer.viewport.contentSize.x;
-        this.imgHeight = this._viewer.viewport.contentSize.y;
+        var contentSizeViewport = {},
+            contentSize;
+        // check if world is defined (OSD 2.2.0 and up)
+        if (this._viewer.world !== undefined){
+          contentSizeViewport.x = this._viewer.world.getHomeBounds().width;
+          contentSizeViewport.y = this._viewer.world.getHomeBounds().height;
+          contentSize = this._viewer.world.getItemAt(0).viewportToImageCoordinates(contentSizeViewport.x, contentSizeViewport.y);
```

Comment:
I assume you're using the first image because this plugin only operates on the first image... is that correct?

Reply:
Btw, I think this won't give the right result if the image isn't positioned at 0,0. I'm not sure if that's an important use case for this plugin. If it is, you could get the x and y of the home bounds and convert them as well and subtract them from the converted width and height.

Reply:
That was my assumption. I think that adding support for multiple images, especially ones which may have different conversion from viewport to image coordinates, would be a bigger fix.